provider/resources.go

```diff
@@ -122,11 +119,6 @@ func Provider() tfbridge.ProviderInfo {
 					EnvVars: []string{"OS_SWAUTH"},
 				},
 			},
-			"use_octavia": {
-				Default: &tfbridge.DefaultInfo{
-					EnvVars: []string{"OS_USE_OCTAVIA"},
-				},
-			},
```

Comment:
This is not present in v2.0.0 of upstream. The pulumi-terraform-bridge v3.83.0 errors on it.